core/src/HIP/Kokkos_HIP.cpp

```diff
@@ -75,15 +75,16 @@ void HIP::impl_initialize(InitializationSettings const& settings) {
   }
 #endif
 #ifdef KOKKOS_ARCH_AMD_GFX942_APU
-  if (!Kokkos::Impl::xnack_enabled()) {
+  if (!Kokkos::Impl::xnack_environment_enabled()) {
     std::cerr << R"warning(
 Kokkos::HIP::initialize WARNING: Could not determine that xnack is enabled.
                                  Kokkos requires xnack to be enabled for
                                  ARCH_AMD_GFX942_APU (MI300A) to access host
                                  allocations from the device. Set HSA_XNACK=1
-                                 in your environment and ensure
-                                 \"CONFIG_HMM_MIRROR=y\" is in the /boot/config
-                                 file and that file is readable)warning"
+                                 in your environment. For further information
+                                 on HMM support call `Kokkos::print_configuration`,
+                                 or run with KOKKOS_PRINT_CONFIGURATION=1 in your
+                                 environemnt.)warning"
```

Comment:
typo environemnt -> environment

Reply:
done in https://github.com/kokkos/kokkos/pull/7765/commits/adc02dd6d769fd477e54fd9bc8896b607e66b86b